support/ruby/collins-state/collins_state.gemspec

```diff
@@ -1,15 +1,11 @@
-# Generated by jeweler
-# DO NOT EDIT THIS FILE DIRECTLY
-# Instead, edit Jeweler::Tasks in Rakefile, and run 'rake gemspec'
 # -*- encoding: utf-8 -*-
 
 Gem::Specification.new do |s|
   s.name = "collins_state"
   s.version = File.read 'VERSION'
 
-  s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.authors = ["Blake Matheny"]
-  s.date = "2012-11-19"
+  s.date = "2015-05-20"
   s.description = "Provides basic framework for managing stateful processes with collins"
   s.email = "bmatheny@tumblr.com"
```

Comment:
We should probably make this a more useful email address, since I doubt anyone is checking Blake's email anymore.